src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -411,12 +412,25 @@ private ImmutableArray<BoundExpression> BuildArgumentsForDynamicInvocation(Analy
         private static bool ReportBadDynamicArguments(
```

Comment:
>ReportBadDynamicArguments [](start = 28, length = 25)

Please add a test for every call-site of ReportBadDynamicArguments when ```in``` arguments are in play.